### ISSUE TITLE
ErrorRecord.invocation_info crash if <Nil N="InvocationInfo" />

### ISSUE THREAD

**Issue by cork** [WinRb/WinRM#309]
If the response InvocationInfo object is empty the ErrorRecord.invocation_info crashes with
```ruby
NoMethodError: undefined method `elements' for nil:NilClass
from /winrm/.bundle/ruby/2.6.0/gems/winrm-2.3.3/lib/winrm/psrp/message_data/error_record.rb:51:in `property_hash'
```

The xml tag in the exception response is:
```xml
<Nil N="InvocationInfo" />
```

**Comment by cork** [comment]
Updated the error_record.rb:53 line number as it was offset by debug prints.

**Comment by mwrock** [comment]
closed by #310
